fdbbackup/backup.actor.cpp

```diff
@@ -165,6 +165,12 @@ enum {
 	OPT_TRACE_FORMAT,
 };
 
+// Top level binary commands.
+CSimpleOpt::SOption g_rgOptions[] = {
```

Comment:
Maybe put `-h` and `--help` here too?

Reply:
Done. What do you think about removing the `-h` and `--help` options from the actions, ie remove `./fdbbackup start --help` in favor of just `./fdbbackup --help`?

Reply:
I think this is a good idea and the printout of the usage needs to be adjusted to reflect the change.

Reply:
I think having `fdbbackup <ACTION> --help` is still useful to see how each action is used. Each action supports different options, so having all action options combined into one large help message can be confusing.

Reply:
I agree. Right now, `fdbbackup <ACTION> --help` is the same for all actions, which is not ideal and confusing.

Reply:
Trevor how much effort do you think this is worth? This will involve some sort of matching between the `CSimpleOpt::SOption` array for the specified action to only print certain options in [`printBackupUsage`](https://github.com/apple/foundationdb/blob/125776491b35b60a1aa03e533faa159351837961/fdbbackup/backup.actor.cpp#L1014).

Reply:
My solution for this is to leave `fdbbackup <ACTION> --help` as is, and add `fdbbackup --help`. More specific changes to what the help text displays on a per action basis are outside the scope of this change.